py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -330,8 +330,13 @@ def determine_resources(ncameras, jobdesc, queue, nexps=1, forced_runtime=None,
     if jobdesc in ('ARC', 'TESTARC'):
         ncores          = 20 * (10*(ncameras+1)//20) # lowest multiple of 20 exceeding 10 per camera
         ncores, runtime = ncores + 1, 45             # + 1 for worflow.schedule scheduler proc
-    elif jobdesc in ('FLAT', 'TESTFLAT'):
+    elif jobdesc in ('FLAT', 'TESTFLAT', 'TILENIGHT'):
         ncores, runtime = 20 * nspectro, 25
```

Comment:
TILENIGHT is enough different from FLAT that I think these should be in separate `elif` blocks to give flexibility to adjust their resource requests independently.

Previously the TILENIGHT case would use 1 node on Perlmutter which I think is what we still want.  Now this is requesting 20*30 cores = 5 Perlmutter nodes here and then later (line 370) setting it back to 1 node, which seems like a fragile code flow.